spec/classes/web_spec.rb

```diff
@@ -117,7 +116,8 @@ def package_provider_for_gems
           database_name: 'zabbix-server',
           database_user: 'zabbix-server',
           database_password: 'zabbix-server',
-          zabbix_server: 'localhost')
+          zabbix_server: 'localhost',
```

Comment:
this will throw:
```ruby
spec/classes/database_mysql_spec.rb:11:21: C: Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.
        facts.merge({ ...
```

the rubocop config in this module is a bit stricter than our default one, and I want to keep that behavior. I will remove the commit as @jyaworski also seems to be fine with that.